.travis.yml

```diff
@@ -0,0 +1,10 @@
+language: python
+python:
+  - "3.7"
```

Comment:
Will be putting this in the project template, but we decided we'll be supporting python 3.6 and 3.8 as well.

Reply:
+1, please add this to travis and the template.